File: 100-Days-Series/Day 58/FloydsTriangle.cpp
#include<iostream>
using namespace std;
// Function to print Floyd' s Triangle
void getPrintFloydsTriangle(int rows){
    int num=1;
    for(int i=1;i<=rows;i++){
        for(int j=1;j<=i;j++){
            cout<<num++<<" ";
        }
        cout<<endl;
    }
}
// Let's Start here
int main(){
    int rows;
    cout<<"\nEnter the number of rows : ";
    cin>>rows;
    getPrintFloydsTriangle(rows);
}

File: 100-Days-Series/Day 58/FloydsTriangle.js

// Functiont to print Floyd's Triangle
const getPrintFloydsTriangle=(rows)=>{
    let num=1;
    for(let i=1;i<=rows;i++){
        line="";
        for(let j=1;j<=i;j++){
            line+=" "+num++;
        }
        console.log(line);
    }
}
// Let's Start here
let rows=5
getPrintFloydsTriangle(rows);